gadgets/traceloop/README.mdx

```diff
@@ -8,38 +8,79 @@ import TabItem from '@theme/TabItem';
 
 # traceloop
 
-The traceloop gadget is a syscalls flight recorder.
+The trace_loop gadget is used to capture and replay system calls in real-time, acting as a flight recorder for your Kubernetes applications.
 
 ## Requirements
 
-- Minimum Kernel Version : 5.10
+- Minimum Kernel Version: 5.10+
+- Dependencies: Inspektor Gadget installed
+- Access: kubectl access to your cluster
 
-## Getting started
+## Getting Started
 
 Running the gadget:
 
 <Tabs groupId="env">
     <TabItem value="kubectl-gadget" label="kubectl gadget">
-        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/traceloop:%IG_TAG% [flags]
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/traceloop:latest [flags]
```

Comment:
The "%IG_TAG%" suffix is automatically replaced when the documentation website is updated. See [version-replacer.js#L10](https://github.com/inspektor-gadget/website/blob/7fbd2de7722dd24199215133a25b4de340d3faba/src/remark/version-replacer.js#L10)

Reply:
Alright then